unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -113,13 +145,13 @@ public async UniTask<FriendRequest> RequestFriendship(string friendUserId, strin
             return new FriendRequest(GetFriendRequestId(friendUserId, createdAt), createdAt, "GET OWN ADDRESS", friendUserId, messageBody);
         }
 
-        private string GetUserIdFromFriendRequestId(string friendRequestId) =>
+        public string GetUserIdFromFriendRequestId(string friendRequestId) =>
             friendRequestId.Split("-")[0];
 
-        private string GetFriendRequestId(string userId, long createdAt) =>
+        public string GetFriendRequestId(string userId, long createdAt) =>
```

Comment:
these methods could be static to avoid needing an instance to use them. Also you can evaluate if you can put together many of these functions and wrap them in a `Utils` library.